src/prefect/server/api/workers.py

```diff
@@ -374,7 +374,7 @@ async def get_scheduled_flow_runs(
 
     polled_work_queue_ids = [wq.id for wq in work_queues]
     ready_work_queue_ids = [
-        wq.id for wq in work_queues if wq.status != WorkQueueStatus.READY
```

Comment:
when `wq.status == WorkQueueStatus.PAUSED`, this is erroneously setting those work queues to READY

Reply:
Should be good. 

https://github.com/PrefectHQ/nebula/blob/ba07d73806589cc5225dda266c6ee02915d19a9f/src/prefect_cloud/orion/api/workers.py#L624-L628

I'm still looking into another closely related issue, but this should fix one bug.